layer4/evaluation_plan.go

```diff
@@ -0,0 +1,164 @@
+package layer4
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
+)
+
+// ChecklistItem represents a single checklist item.
+type ChecklistItem struct {
+	// RequirementId is the requirement ID (e.g., "AC-1.1")
+	RequirementId string
+	// ProcedureName is the procedure name
+	ProcedureName string
+	// Description is the description (if different from name)
+	Description string
+	// Documentation is the documentation URL
+	Documentation string
+	// IsAdditionalProcedure indicates if this is an additional procedure
+	IsAdditionalProcedure bool
+}
+
+// ControlSection organizes checklist items by control.
+type ControlSection struct {
+	// ControlName is the control identifier (e.g., "AC-1")
+	ControlName string
+	// ControlReference is the formatted reference (e.g., "OSPS-B / AC-01")
+	ControlReference string
+	// Items are the checklist items for this control
+	Items []ChecklistItem
+}
+
+// Checklist represents the structured checklist data.
+type Checklist struct {
+	// PlanId is the evaluation plan ID
+	PlanId string
+	// Author is the author name
+	Author string
+	// AuthorVersion is the author version
+	AuthorVersion string
+	// Sections are the control sections
+	Sections []ControlSection
+}
+
+// ToChecklist converts an EvaluationPlan into a structured Checklist.
+func (e EvaluationPlan) ToChecklist() Checklist {
+	checklist := Checklist{}
+
+	if e.Metadata.Id != "" {
+		checklist.PlanId = e.Metadata.Id
+	}
+	if e.Metadata.Author.Name != "" {
+		checklist.Author = e.Metadata.Author.Name
+		checklist.AuthorVersion = e.Metadata.Author.Version
+	}
+
+	for _, plan := range e.Plans {
+		if plan.Control.EntryId == "" {
+			continue
+		}
+
+		// Get control name with fallback: EntryId -> ReferenceId -> default
+		controlName := "Unnamed Control"
+		if plan.Control.EntryId != "" {
+			controlName = plan.Control.EntryId
+		} else if plan.Control.ReferenceId != "" {
+			controlName = plan.Control.ReferenceId
+		}
+
+		// Format control reference as "Framework / Control-ID" (e.g. OSPS-B / AC-01)
```

Comment:
Just a nit, for clarity using a real control id.
```suggestion
		// Format control reference as "Framework / Control-ID" (e.g. OSPS-B / OSPS-AC-01)
```